examples/nlp/language_modeling/tuning/conf/megatron_gpt_sft.yaml

```diff
@@ -71,6 +71,11 @@ model:
 
   data:
     chat: False # whether use chatbot data or not
+    chat_prompt_tokens:  # special tokens for the chat prompts, a dictionary of {token_type: token}. note that some tokenizer may combine the characters at the junction between {end_of_turn}{turn_start}. e.g. '<im end><im start>', the '><' sometimes is merged to be a single token. This is not supported, try to avoid
```

Comment:
Maybe a good thing to mention is what these should be for llama2? Otherwise, people might not know what to set it to

Reply:
llama2 can just use the default. it is user's choice to do whatever here.